packages/flutter/test/material/refresh_indicator_test.dart

```diff
@@ -1178,29 +1169,78 @@ void main() {
         ),
       );
 
-      RefreshProgressIndicator getIndicator() {
-        return tester.widget<RefreshProgressIndicator>(
-          find.byType(RefreshProgressIndicator),
-        );
-      }
-
       // Correspond to [_kDragContainerExtentPercentage].
       final double maxPosition = tester.view.physicalSize.height / tester.view.devicePixelRatio * 0.25;
-      for (final double position in positions) {
+      for (final double position in <double>[50.0, 100.0, 150.0]) {
         await tester.fling(find.text('X'), Offset(0.0, position), 1.0);
         await tester.pump();
         positionController.value = position / maxPosition;
         expect(
-          getIndicator().valueColor!.value!.alpha,
+          tester.widget<RefreshProgressIndicator>(find.byType(RefreshProgressIndicator)).valueColor!.value!.alpha,
           valueColorAnimation.value!.alpha,
         );
         // Wait until the fling finishes before starting the next fling.
         await tester.pumpAndSettle();
       }
     }
+  });
+
+  testWidgets('RefreshIndicator default elevation value correctly', (WidgetTester tester) async {
+    final AnimationController positionController = AnimationController(vsync: const TestVSync());
+    addTearDown(positionController.dispose);
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: RefreshIndicator(
+          onRefresh: refresh,
+          child: ListView(
+            physics: const AlwaysScrollableScrollPhysics(),
+            children: const <Widget>[Text('X')],
+          ),
+        ),
+      ),
+    );
+
+    // Correspond to [_kDragContainerExtentPercentage].
+    final double maxPosition = tester.view.physicalSize.height / tester.view.devicePixelRatio * 0.25;
+    for (final double position in <double>[50.0, 100.0, 150.0]) {
```

Comment:
Is all this flinging really necessary, or can you just do the expect once and be done?

Reply:
Not necessary, but again i am referring test-cases from above. 

I believe one will suffice as well.